src/main/scala/com/feedzai/cosytest/DockerComposeJavaSetup.scala

```diff
@@ -62,31 +63,110 @@ case class DockerComposeJavaSetup(
     *
     * @return true if setup network and containers were removed with success otherwise false
     */
-  def cleanUp: Boolean = dockerSetup.cleanUp()
+  def cleanup: Boolean = dockerSetup.cleanup()
 
+  /**
+    * Returns the port open in localhost mapped to container port.
+    *
+    * @param containerId
+    * @param port exposed from container
+    * @return string of open localhost port. Empty if a failure occurs
+    */
   def getContainerMappedPort(containerId: String, port: Int): String =
     dockerSetup.getContainerMappedPort(containerId, port)
 
+  /**
+    * Returns the list of container Ids associated to service.
+    *
+    * @param serviceName string
+    * @return list of container ids. If a failure occurs an empty list is returned
+    */
   def getServiceContainerIds(serviceName: String) =
     new util.ArrayList[String](dockerSetup.getServiceContainerIds(serviceName).asJava)
 
+  /**
+    * Returns the list of container IPs associated to service.
+    *
+    * @param serviceName string
+    * @return list of container ips.
+    */
+  def getServiceContainerIps(serviceName: String) =
```

Comment:
I spent a bit of time trying to find the difference from the previous method :) What about `getServiceContainerAddresses`?